### ISSUE TITLE
use FQDN in camel TypeConverter file

### ISSUE THREAD

**Issue by jludvice** [czgov/camel-isds#34]
http://camel.apache.org/type-converter.html

file `src/main/resources/META-INF/services/org/apache/camel/TypeConverter`
allows to specify just package name or FQDN
but package name doesn't work with spring-boot and openshift.
Need to switch to FQDN.

Following exception is thrown unless we specify FQDN
```
Caused by: org.apache.camel.TypeConverterLoaderException: Failed to load type converters because of: Cannot find any type converter classes from the following packages: [com.github.czgov.isds.types]
	at org.apache.camel.impl.converter.AnnotationTypeConverterLoader.load(AnnotationTypeConverterLoader.java:119)
	at org.apache.camel.impl.converter.BaseTypeConverterRegistry.loadTypeConverters(BaseTypeConverterRegistry.java:602)
	at org.apache.camel.impl.converter.DefaultTypeConverter.doStart(DefaultTypeConverter.java:53)
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61)
	at org.apache.camel.util.ServiceHelper.startService(ServiceHelper.java:75)
	at org.apache.camel.impl.DefaultCamelContext.doAddService(DefaultCamelContext.java:1283)
	at org.apache.camel.impl.DefaultCamelContext.addService(DefaultCamelContext.java:1243)
	at org.apache.camel.impl.DefaultCamelContext.getTypeConverter(DefaultCamelContext.java:2369)
	... 38 common frames omitted
```

**Comment by jludvice** [comment]
Fix (161fb16f3166b75ac73928aa3726ef726eb48034) released in version 0.2.0.

**Comment by jludvice** [comment]
There is issue in camel issue tracker to include note about this in camel docs:
https://issues.apache.org/jira/browse/CAMEL-11371
